mautic/mautic
# Updated location fields for empty location.

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [5.x]
| New feature/enhancement? (use the a.x branch)      | [ :x: ]
| Deprecations?                          | [ :x: ]
| BC breaks? (use the c.x branch)        | [ :x: ]
| Automated tests included?              | [ ✅ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #13720  



#### Description:
If a contact has no address, the listing shows a comma sign (,) in the Location column for that contact.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Navigate to the contacts page
3. Create quick contact without adding the State, City, and Country
4. Go back to the listing page.
